.github/workflows/check-doc-updates.yml

```diff
@@ -0,0 +1,15 @@
+name: Check for updates in Documentation index file
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check-for-updates:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check for updates in Documentation index file
+        run: ./scripts/automation.py update --check
```

Comment:
We tend to have all check scripts via Makefile, so I would wrap this in a Makefile and then call the make doc-check here or something similar.

Reply:
If the include solution does not work, I will add it here

Reply:
It did not work with `mkdocs serve` (and I believe it would not work with the README file as well). I will add it to makefile in the next commit

![include_error](https://user-images.githubusercontent.com/66965232/194867800-d8ed1cb0-d764-47b8-9d88-e482bb365aaa.png)